src/react/src/components/Contribute/ProductionLocationDialog.jsx

```diff
@@ -71,15 +71,13 @@ const getStatusBadgeClass = (classes, status) => {
     }
 };
 
-const getTooltipText = (claimStatus, moderationStatus) => {
-    if (moderationStatus === MODERATION_STATUSES_ENUM.PENDING) {
-        return "You'll be able to claim the location after the moderation is done.";
-    }
-
+const getTooltipText = claimStatus => {
     if (claimStatus === PRODUCTION_LOCATION_CLAIM_STATUSES_ENUM.CLAIMED) {
         return 'Production location has been claimed already.';
     }
-
+    if (claimStatus === PRODUCTION_LOCATION_CLAIM_STATUSES_ENUM.PENDING) {
+        return 'A claim for this production location is pending approval.';
+    }
     return 'Claim is not available.';
```

Comment:
It is probably not related to this task. Was it approved that this should be this specific text? In the design, I see a different one.
<img width="674" alt="Screenshot 2025-02-12 at 10 07 24" src="https://github.com/user-attachments/assets/276411a4-bfbe-41b1-8651-b1e746d46481" />

Reply:
Fixed, thank you!